internal/pkg/pkg.go

```diff
@@ -36,8 +36,12 @@ const ParentDir = ".."
 
 // Pkg represents a kpt package with a one-to-one mapping to a directory on the local filesystem.
 type Pkg struct {
-	UniquePath  types.UniquePath
-	DisplayPath types.DisplayPath
+	UniquePath        types.UniquePath
```

Comment:
document semantics of the exported fields here, especially since you're changing DisplayPath

Reply:
Updated.